dts/bindings/sensor/ti,bq274xx.yaml

```diff
@@ -31,6 +31,13 @@ properties:
     required: true
     description: Battery Terminate Voltage in mV
 
+  chemistry-id:
+    type: int
+    default: 0
```

Comment:
The description needs to provide a justification for the default. Is it a register reset value, for example?

Reply:
I slightly changed the description, I'm not entirely sure it helps. I needed a sentinel value, as you may not want to check the register in case you want to just assume the correct IC variant of the BQ27421 has been selected to match the battery on the board.

I chose zero, as it is not used as a chemistry ID. I could go with -1 to be fully outside the range of possible values.

Reply:
I suggest removing the default from here, and adding it in BQ274XX_INIT.